dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataImport.java

```diff
@@ -785,16 +785,18 @@ private void buildRelationObject(Context c, Item item, String value, boolean lef
      *                                          for the right entity
      * @param relationshipType                  The RelationshipType object that we want to check whether it's
      *                                          valid to be added or not
-     * @return                                  A boolean indicating whether the relationship is left or right, will
+     * @return A boolean indicating whether the relationship is left or right, will
      *                                          be false in this case
      * @throws SQLException                     If something goes wrong
      */
-    private boolean handleRightLabelEqualityRelationshipTypeElement(Context c, Entity entity, Entity relationEntity,
-                                                                    boolean left,
-                                                                    List<RelationshipType> acceptableRelationshipTypes,
-                                                                    List<RelationshipType>
-                                                                        rightRelationshipTypesForEntity,
-                                                                    RelationshipType relationshipType)
+    private boolean handleRightwardTypeEqualityRelationshipTypeElement(Context c, Entity entity,
```

Comment:
It feels as though this code has too many jobs.  It wants to be named `isLeftRelationship`, but it also wants to be named `addRightRelationship`.  Or something like that.  It can only be understood in context, which suggests to me that the boundary between this method's function and the caller's is in the wrong place.

"handle" is something done with exceptions or interrupts.

Reply:
I've renamed it to verifyValidRightwardRelationshipType
Combined with the JavaDocs I hope this clarifies the method's purpose